src/deployment/deploy.py

```diff
@@ -1238,7 +1304,14 @@ def main() -> None:
         nargs="*",
         help="Set additional AAD tenants beyond the tenant the app is deployed in",
     )
-
+    parser.add_argument(
+        "--enable_dotnet",
```

Comment:
You should be able to parse the functions to a list instead of a string. check `--apply_migrations`.  But it might change the syntax. 
Feel free to ignore this suggestion.

Reply:
Makes sense to keep the argument inputs uniform and this wouldn't require changing much of the previous code.